### DISCUSSION TITLE
how can we add or remove existing anime attributes

### DISCUSSION THREAD

**Discussion by neonarc4** [juliangarnier/anime#888] [Q&A] [answered]
since it not timeline it cant add or update existing divanimation  
divAnimation.add({'scale':[0,1]})

**Comment by neonarc4** [comment]
@juliangarnier help bro :( 

i need to modify the anime do i need to recreate everytime something like this 
```js

let divAnimation = anime({
targets: "panda",
translateX: 300,
elasticity: 200,
easing: 'easeInOutCubic',
autoplay: false,
opacity: 0.7
});
divAnimation = anime({
.....same above just add scale also target same 
scale:[0,1],
});
```
is it possible to achieve without the timeline

**Comment by juliangarnier** [comment]
Hey sorry I don't really understand, what do you mean by "anime attributes" ?
Can you be more clear on what exactly you are trying to achieve?

**Reply by neonarc4** [reply]
something like to have different animation on breakpoints 

like how can i modify or add scale in current divAnimation of animejs  /// like short of patch in axios when we update certain portion of data 

i mean by anime attributes updating the existence once 
 ```

targets: "panda",
translateX: 300,
elasticity: 200,
easing: 'easeInOutCubic',
autoplay: false,
opacity: 0.7
```

**Reply by juliangarnier** [reply]
> something like to have different animation on breakpoints

In V3 there is no built-in way to handle responsive animation (if it's what you mean by "breakpoint"), but you can use a mix of [`matchMedia()`](https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia) and `animation.remove(target)` to handle different animation for different media queries.

In V4 you'll be able to use `createScope()` to easily handle different animations at different breakpoints.

> also brother is their any reverse approach for this element.seek((nv / 25) / element.duration); ?
> 
> like i calculate % of an scroll height but the issue is like if i go at 29 then i can no longer make it reverse seek ? like after 29 i want to make a reverseseek()

Please one question per thread, and provide an example of what you trying to achieve a clear description of the issue with a live example.

Thanks.

**Reply by neonarc4** [reply]
julian sorry this one is last help :( plz  
will try to purchase next month gonna check out scope bit learning basic before v4 >.< 
https://codepen.io/haideraf/pen/abxPxQO 
this one i really want ur help how can i reverse the process after 50% of scroll

**Reply by juliangarnier** [reply, marked as answer]
In order to correctly update an animation when a new breakpoint is reached, you need a way to correctly cancel it and revert all the inline styles applied to the elements by the animation, then create the new animation with the breakpoint specific parameters. V3 doesn't have these features built-in, but it's going to be available in V4 (it's currently in the beta branch).

**Reply by neonarc4** [reply]
was doing it like having a new animejs to 0 1 and 1 to 0 reverse it but thnx for bringing it on v4 might be much easier and better approach

**Reply by juliangarnier** [reply]
You need to calculate the scroll position, and when reaching 50%, call `divAnimation.reverse()`.
V4 has an `onScroll` method that will help with scroll based animations!

**Reply by neonarc4** [reply]
just did an direction : 'reverse' after reaching 50% but yah .reverse() good thnx alot >.< /
